domains/fishy.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "<skibidiloser>",
+    "email": "<auroralindgren23@gmail.com>"
+  }
+}
+{
+  "record": {
+    "CNAME": "<skibidiloser>.github.io"
```

Comment:
```suggestion
    "CNAME": "skibidiloser.github.io"
```

Reply:
Also site gives 404 error, if you have the site hosted then please send a preview of the website